src/types/ErrorMessage.ts

```diff
@@ -0,0 +1,7 @@
+export enum ErrorMessage {
+  NONE = '',
+  LOAD = 'Unable to load a todo',
+  ADD = 'Unable to add a todo',
+  DELETE = 'Unable to delete a todo',
+  UPDATE = 'Unable to apdate a todo',
```

Comment:
There is a typo in the description of the UPDATE error message. Change 'apdate' to 'update' 📝